examples/eth1_with_sriov.yml

```diff
@@ -5,6 +5,7 @@ interfaces:
     state: up
     ethernet:
       sr-iov:
+        autoprobe-drivers: false
```

Comment:
The kernel name for this is `sriov_drivers_autoprobe`, instead of using NetworkManager name, can we try `drivers_autoprobe`?

Reply:
I have changed it to better match the kernel name.